monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -821,6 +838,23 @@ object Observable {
   def concatDelayError[A](sources: Observable[A]*): Observable[A] =
     Observable.fromIterable(sources).concatMapDelayError[A](identity)
 
+  /**
+    * like `zipWith2`, but when `oa1` completes before `oa2` does, the new observable will
+    * continue emitting items as `f(paddingA1, oa2.onNext())` until `oa2` completes;
+    * and vice versa, when `oa2` completes before `oa1` does, the new observable will
+    * continue emitting items as `f(oa1.onNext(), paddingA2)` until `oa1` completes.
+    */
+  def pad2With[A1,A2,R](oa1: Observable[A1], oa2: Observable[A2], paddingA1: ⇒ A1, paddingA2: ⇒ A2)(f: (A1,A2) ⇒ R): Observable[R] =
+    new builders.Pad2Observable(oa1, oa2, paddingA1, paddingA2)(f)
+  /**
+    * like `zip2`, but when `oa1` completes before `oa2` does, the new observable will
```

Comment:
This description needs a title.

Also `@param` descriptions in this case would be useful I think, even if a little redundant with the description.